packages/website/src/components/breadcrumbs.tsx

```diff
@@ -27,7 +27,7 @@ const NextLink = React.forwardRef(
     ref: React.Ref<HTMLAnchorElement>,
   ) => {
     return (
-      <Link href={href}>
+      <NavLink to={href}>
         <a
```

Comment:
react-router doesn't want to populate a child anchor the way next link does, so we'll need to pull out these nested anchors. It would be good to do this outside of the usages of link so we have an easy code path to switch backwards and forwards between.